widget/src/progress_bar.rs

```diff
@@ -157,6 +157,17 @@ where
                 style.bar,
             );
         }
+
+        if style.border.color != Color::TRANSPARENT {
+            renderer.fill_quad(
+                renderer::Quad {
+                    bounds: Rectangle { ..bounds },
+                    border: style.border,
+                    ..renderer::Quad::default()
+                },
+                Color::TRANSPARENT,
+            );
+        }
```

Comment:
We should probably set the border color of the base quad as transparent, or we may run into issues with translucent borders.

Reply:
Yeah, I originally did that at first, but when resizing the window, I would sometimes see really thin gaps between the base quad and the border:
![Screenshot 2024-05-20 131211](https://github.com/iced-rs/iced/assets/40839054/0e3ff2ec-5bd5-4924-8669-8aec29be7ec5)

Though this is really an issue with `wgpu` since this doesn't happen on `tiny-skia`.

Nonetheless, I found a better solution to all this. I can instead include the border information to the active bar but make the border colour transparent:

![Screenshot 2024-05-20 132308](https://github.com/iced-rs/iced/assets/40839054/00b5001d-aa4d-4739-b8c2-2418a577fdc9)

Though annoying gap is still there, but this time it's between the active bar and background. But I'll take it:

![Screenshot 2024-05-20 132313](https://github.com/iced-rs/iced/assets/40839054/21bb970c-73ce-474c-bd5a-188ffd910eba)